### PR TITLE
[SPARK-31418][CORE][FOLLOW-UP][MINOR] Fix log messages to print stage id instead of the object name

### PR DESCRIPTION
**Pull request by venkata91**
### What changes were proposed in this pull request?
Just few log lines fixes which are logging the object name instead of the stage IDs

### Why are the changes needed?
This would make it easier later for debugging.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Just log messages. Existing tests should be enough

### CODE REVIEW

**Comment by venkata91** [comment]
@tgravescs This is just a minor log line changes which I found printing object names instead of the IDs.

**Comment by tgravescs** [comment]
ok to test

**Comment by tgravescs** [comment]
what did the log look like before? I thought printing the taskSet should give you something like: TaskSet 2.0. where 2 is the stage id and 0 is the stage attempt id. Now its only printing the stage id without the attempt id?

**Comment by venkata91** [comment]
Infact its not even printing stageId rather printing the TaskSetManager object name. Do you think instead of printing stageId its better to print name `taskSet.name`?

For eg: ```20/07/29 16:10:04 INFO YarnScheduler: Notifying ExecutorAllocationManager to allocate more executors to schedule the unschedulable task before aborting org.apache.spark.scheduler.TaskSetManager@f031375.

20/07/29 16:10:04 INFO YarnScheduler: Waiting for 120000 ms for completely blacklisted task to be schedulable again before aborting org.apache.spark.scheduler.TaskSetManager@f031375.

**Comment by tgravescs** [comment]
what if you explicitly put ${taskSet.toString}

**Comment by tgravescs** [comment]
oh sorry, I was looking at the wrong thing, those are TaskSetManagers not TaskSet

**Comment by venkata91** [comment]
> oh sorry, I was looking at the wrong thing, those are TaskSetManagers not TaskSet

Yes thats right. I think `${taskSet.stageId}` would be good enough. Though I tried with `${taskSet.name}` which prints like this:
```20/07/29 18:49:31 INFO YarnScheduler: Waiting for 120000 ms for completely blacklisted task to be schedulable again before aborting TaskSet_0.0.```

With stageId:

```20/07/29 18:52:44 INFO YarnScheduler: Notifying ExecutorAllocationManager to allocate more executors to schedule the unschedulable task before aborting stage 0.
20/07/29 18:52:44 INFO YarnScheduler: Waiting for 120000 ms for completely blacklisted task to be schedulable again before aborting stage 0.```

Or do you think its better to add `toString` to `TaskSetManager`?

**Comment by tgravescs** [comment]
no, I think this is fine.

**Review by tgravescs — APPROVED** [review]

File: core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala [hunk 1]
```diff
@@ -663,15 +663,16 @@ private[spark] class TaskSchedulerImpl(
                   if (Utils.isDynamicAllocationEnabled(conf)) {
                     if (!unschedulableTaskSetToExpiryTime.contains(taskSet)) {
                       logInfo(s"Notifying ExecutorAllocationManager to allocate more executors to" +
-                        s" schedule the unschedulable task before aborting $taskSet.")
+                        s" schedule the unschedulable task before aborting" +
```
Comment (maropu) [inline core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala:666]: nit: remove `s` in the head.

**Comment by HyukjinKwon** [comment]
retest this please

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
retest this please

**Comment by HyukjinKwon** [comment]
Merged to master.

**Pull request closed without merging**